bmc-acf/acf_manager.cpp

```diff
@@ -394,7 +397,21 @@ acf_info ACFCertMgr::installACF(std::vector<uint8_t> accessControlFile)
         }
         else
         {
-            log<level::ERR>("cannot read production key file");
+            log<level::ERR>("cannot read production backup key file");
+            elog<InternalFailure>();
```

Comment:
nit: InternalFailure is something we try to avoid in the community. Also, this path doesn't seem related to the new prodBackup2KeyExists variable? Wont an InternalFailure also throw and unhandled exception which means we don't try the new backup below?

Reply:
This really is an internal error.  The code above already check if the public key file exists, so it should be readable.  I've never seen this happen in practice.  And I don't want to change the control flow.  So I didn't change it.